source/Subsystem/Renderer/RenderContext.h

```diff
@@ -14,13 +15,15 @@ class RenderContext
 
 	void Initialize();
 	void CreateBuffers(std::vector<Model>& aModels);
-	void Render(const std::vector<Model>& aModels, const TextureLibrary& aTextureLibrary, const ShaderLibrary& aShaderLibrary, int aWidth, int aHeight, double aDeltaTime);
+	void Render(const std::vector<Model>& aModels, const TextureLibrary& aTextureLibrary, ShaderLibrary& aShaderLibrary, int aWidth, int aHeight, double aDeltaTime);
 	void Destroy(const std::vector<Model>& aModels);
 
 private:
 	void PrintDebugInfo();
 	void CreateCamera();
+	void CreateLight();
 
 private:
 	Camera* myCamera;
+	Light* myLight;
```

Comment:
This hurts my soul! Could be fixed later tho

Reply:
What would you suggest? Right now I just made a raw pointer to test lighting.

Reply:
I was referring to the single light stored in the render context... Should be handled by world or wherever scenes are stored. Should be fine for now till we expand on the lighting system!

Reply:
Yes, I agree. We'll probably do that soon.

Reply:
@JasperDre call me UwU